app/views/user/index.html.erb

```diff
@@ -1 +1,25 @@
-<h1>User Index HTML File</h1>
\ No newline at end of file
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>"Blog Website"</title>
+    <%= stylesheet_link_tag "user/index",
+    media: "all",
+    "data-turbolinks-track": "reload" %>
+  </head>
+  <body>
+
+    <div class="subParentContent">
+       <% @users.each do |user| %>
+      <div class="contentIconName">
+        <div class="contentIcon">
+          <%= image_tag "icon.png", alt: "adadasda", class: "contentIcon" %>
```

Comment:
- [ ] It would be better if we have a `user.photo` attribute in our database or model like what we have currently with `user.name` and `user.post_counter`
![Screenshot from 2023-02-03 00-32-38](https://user-images.githubusercontent.com/46388113/216481322-594dc835-f2e8-4dc5-bc73-b52d32211a3e.png)